examples/imagenet/train.py

```diff
@@ -236,10 +234,15 @@ def create_train_state(rng, config: ml_collections.ConfigDict,
     dynamic_scale = None
 
   params, model_state = initialized(rng, image_size, model)
-  optimizer = optim.Momentum(
-      beta=config.momentum, nesterov=True).create(params)
-  state = TrainState(
-      step=0, optimizer=optimizer, model_state=model_state,
+  tx = optax.chain(
```

Comment:
Can we use optax's `alias.sgd` instead?

https://github.com/deepmind/optax/blob/master/optax/_src/equivalence_test.py#L103

Reply:
Ah cool, didn't see the recent update to directly use schedules in the aliases yet.

Reply:
This is quite a bit cuter, but less instructional. I can see advantages with both approaches.